storage/badger/operation/common.go

```diff
@@ -65,7 +65,7 @@ func insert(key []byte, entity interface{}) func(*badger.Txn) error {
 			return storage.ErrAlreadyExists
 		}
 		if err != badger.ErrKeyNotFound {
-			return fmt.Errorf("could not check key: %w", err)
+			return storage.ErrNotFound
```

Comment:
The `update` method [converts the `badger.ErrKeyNotFound` error into `storage.ErrNotFound`](https://github.com/onflow/flow-go/pull/700/files#diff-7a506b5faa037c4ce231b617d70fd28e27fa65d7957cc924ff1674b43479e148R94-R96), whereas the `insert` function here still return `badger.ErrKeyNotFound`, this is inconsistent.

Rather, we could just make it consistent to return `storage.ErrNotFound`, so that the rule is now simple:

if an error is returned by the badger API, then the not found error is `badger.ErrKeyNotFound`, if an error is returned by our storage badger module, then it's `storage.ErrNotFound`